io.github.mgerhardy.vengi.yml

```diff
@@ -0,0 +1,64 @@
+app-id: io.github.mgerhardy.vengi
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: vengi-voxedit
+rename-icon: voxedit-icon
+rename-desktop-file: vengi-voxedit.desktop
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+  cleanup:
+    - /include
+    - /share/man
+    - /lib/pkgconfig
+
+  modules:
+  - name: opencl-headers
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/include/CL
+      - cp -av CL/* /app/include/CL
+    sources:
+      - type: archive
+        url: https://github.com/KhronosGroup/OpenCL-Headers/archive/refs/tags/v2022.01.04.tar.gz
+        sha256: 6e716e2b13fc8d363b40a165ca75021b102f9328e2b38f8054d7db5884de29c9
+
+  - name: libdwarf
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/davea42/libdwarf-code/
+        tag: v0.4.1
+        commit: 9416aa9452c37b3ac252e709c39b46705916b97e
+
+  - name: vengi
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DUNITTESTS=OFF
+      - -DVISUALTESTS=OFF
+      - -DGAMES=OFF
+      - -DMAPVIEW=OFF
+      - -DAIDEBUG=OFF
+      - -DVOXCONVERT=OFF
+      - -DTHUMBNAILER=OFF
```

Comment:
Just curious, why are these being disabled?

Reply:
Because it won't work inside the sandbox environment.